ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedgerSpec.scala

```diff
@@ -98,9 +108,13 @@ class SqlLedgerSpec
         )
 
       } yield {
-        ledger1.ledgerId should not be equal(LedgerId)
+        ledger1.ledgerId shouldBe ledgerId
```

Comment:
This inverts the check, is this intended?

Reply:
notice that the previous check had `LedgerId` with a capital `L`, referring to "companion object" of the type `LedgerId`, thus this check was always true (and therefore useless).